test/benchmark/main.cpp

```diff
@@ -130,7 +130,7 @@ static void BM_Patch(benchmark::State& state)
     for (auto _ : state)
     {
         HTTPRequest::instance().patch(
-            HttpURL("http://localhost:44441/"), R"({"foo": "bar"})"_json, [&](const std::string& /*result*/) {});
+            RequestParameters {.url = HttpURL("http://localhost:44441/12345"), .data = R"({"foo": "bar"})"_json});
```

Comment:
Question: Was the addition of `12345` on purpose? 
```suggestion
            RequestParameters {.url = HttpURL("http://localhost:44441/12345"), .data = R"({"foo": "bar"})"_json});
```

Reply:
Good catch!
Copy-paste mistake, fixed.